pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -1005,6 +1005,15 @@
     "vendorSha256": null,
     "version": "0.14.5"
   },
+  "snowflake": {
+    "owner": "chanzuckerberg",
+    "provider-source-address": "registry.terraform.io/chanzuckerberg/snowflake",
+    "repo": "terraform-provider-snowflake",
+    "rev": "v0.27.0",
+    "sha256": "sha256-myzsEQuErItdjlm8nQgy4n7TlRYyqR26SDF7B4woudY=",
+    "vendorSha256": "sha256-hSt9M16igpMM7/Hl5eJYz3tgat6JeJ5O45RDn1NsZ+I=",
+    "version": "0.27.0"
```

Comment:
Version on the registry is older than this, it will get downgraded automatically.

Reply:
Thanks! Will downgrade.